.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -0,0 +1,39 @@
+---
+name: Bug report
+about: Create a bug report to help improve the ONNX.
+title: ''
+labels: 'bug'
+assignees: ''
+
+---
+# Bug Report
+If the model conversion is failing for a **tutorial** in this repo, report the bug here. However, if the bug is related to general model conversion, please go to the appropriate converter repo.
```

Comment:
It might be useful to also point folks towards the onnx/onnx repo for questions about specific operator functionality. Likewise, onnx/models for questions related to models in the Model Zoo. What do you think @askhade?

Reply:
I think it is ok to not add this. Guidance for model conversion is added specifically because it falls under converters, and the expertise required to offer help\resolve bugs is more there.